Tests/MountebankSwiftTests/Models/Stub/Response/ResponseTests.swift

```diff
@@ -0,0 +1,120 @@
+import MountebankSwift
+
+import XCTest
+
+final class ResponseTests: XCTestCase {
+    func testText() throws {
+        try assertEncode(
+            Is.Examples.text.value,
+            Is.Examples.text.json
+        )
+        try assertDecode(
+            Is.Examples.text.json,
+            Is.Examples.text.value
+        )
+    }
+
+    func testHtml() throws {
+        try assertEncode(
+            Is.Examples.html.value,
+            Is.Examples.html.json
+        )
+        try assertDecode(
+            Is.Examples.html.json,
+            Is.Examples.html.value
+        )
+    }
+
+    func testJson() throws {
+        try assertEncode(
+            Is.Examples.json.value,
+            Is.Examples.json.json
+        )
+        try assertDecode(
+            Is.Examples.json.json,
+            Is.Examples.json.value
+        )
+    }
+
+    func testCodable() throws {
+        try assertEncode(
+            Is.Examples.jsonEncodable.value,
+            Is.Examples.jsonEncodable.json
+        )
+        // Not possibel to decode json back into codable
```

Comment:
Typo: 
`// Not possible to decode json back into codable`

Reply:
Thanks